docker/addons/cassandra-writer/subjects.toml

```diff
@@ -0,0 +1,4 @@
+# If you want to listen on all subjects, just pass one element ["channel.>"], otherwise
+# pass the list of subjects.
```

Comment:
As I mention, an example of subjects, for example one with and one without subtopic should be added.

Example: `["channel.<channel_ID>.messages, channel.<channel_id>.messages/sub/topic/x, ...]`

Reply:
Something like below?
```
# If you want to listen on all subjects, just pass one element [">"], otherwise
# pass the list of subjects (e.g ["channel.<channel_ID>.messages", "channel.<channel_id>.messages/sub/topic/x", ...]).
```

Reply:
`channel.<channel_id>.messages/sub/topic/x` should be `channel.<channel_id>.messages.sub.topic.x`. Also - do we use `messages` on NATS? Maybe, I am not sure, needs to be checked (I know it is used in adapter API, but I am not sure for NATS).

Reply:
@drasko You are right, each topic are separated by dots. You are also right about `messages`. The explanation in PR of documentation repository is correct. The edited comment is:
```
# If you want to listen on all subjects, just pass one element [">"], otherwise
# pass the list of subjects (e.g ["channel.<channel_ID>", "channel.<channel_id>.sub.topic.x", ...]).
```
If it is OK for you, I will edit four `subjects.toml` files.